src/cpp/src/gguf_utils/building_blocks.cpp

```diff
@@ -803,15 +725,15 @@ ov::Output<ov::Node> make_rms_norm(
                 }
             }
         } else if (weight_tensor.get_element_type() == ov::element::f16) {
-            const uint16_t* data = weight_tensor.data<uint16_t>();
+            const uint16_t* data = weight_tensor.data<uint16_t>(), one_fp16 = 0x3C00;
```

Comment:
I thought `one_fp16` is a pointer until I saw the removed TODO. Can you split in two lines add a comment?

Reply:
split lines.
Which comment is required? I suppose the code is self-explainable now